### PR TITLE
Resolve #150

### LINKED ISSUE #150: Threads List Queries Optimization
On the Threads list page, we should not check for subscription, for an anonymous user.

Is there any way to minimize the queries.

threads-list-queries

### PR DESCRIPTION
**Pull request by arifkhn46**
Skip subcription query if users is anonymous

### CODE REVIEW

**Comment by arifkhn46** [comment]
I don't know why styleci is  failing.

**Comment by Indemnity83** [comment]
You need a single empty line (no spaces or tabs) before the return statement.

File: app/Thread.php [hunk 1]
```diff
@@ -195,6 +195,9 @@ public function subscriptions()
      */
     public function getIsSubscribedToAttribute()
     {
+        if (! auth()->id()) {
+            return false;
+        }
         return $this->subscriptions()
```
Comment (Indemnity83) [inline app/Thread.php]: Add a empty line before this return statment.

Comment (arifkhn46) [reply app/Thread.php]: I added the empty line but it still failing :(

**Comment by Indemnity83** [comment]
You still have a space in the newline, maybe the easier thing is to just pull in the diff from styleci

```bash
wget https://styleci.io/analyses/zeKWE7/diff 
git apply diff
```

Its not really critical; Jeffrey can fix it; was just trying to help out.

**Comment by arifkhn46** [comment]
Thanks!

**Pull request merged**